packages/cw-paginate/src/lib.rs

```diff
@@ -85,6 +85,29 @@ where
     }
 }
 
+/// Same as `paginate_map` but only returns the values.
+pub fn paginate_map_values<'a, K, V>(
+    deps: Deps,
+    map: &Map<'a, K, V>,
+    start_after: Option<K>,
+    limit: Option<u32>,
+    order: Order,
+) -> StdResult<Vec<V>>
+where
+    K: Bounder<'a> + KeyDeserialize<Output = K> + 'static,
+    V: serde::de::DeserializeOwned + serde::Serialize,
+{
+    let items = map
```

Comment:
Need to patch with @onewhiskeypls's fix here: https://github.com/DA0-DA0/dao-contracts/pull/447#discussion_r944961140